pyproject.toml

```
@@ -31,7 +31,7 @@ dependencies = [
   "frequenz-api-dispatch @ git+https://github.com/frequenz-floss/frequenz-api-dispatch.git@223315c",
 #  "frequenz-api-dispatch >= 0.13.0, < 0.14",
   # Directly use unreleased commit until the next release
-  "frequenz-client-base @ git+https://github.com/frequenz-floss/frequenz-client-base-python.git@57d3093",
+  "frequenz-client-base @ git+https://github.com/frequenz-floss/frequenz-client-base-python.git@03f2ed2",
```

There is a release available now, so you could use:
```suggestion
  "frequenz-client-base >= 0.2.1, < 0.3.0",
```